controllers/accounts.js

```diff
@@ -109,99 +98,92 @@ router.get('/search' , function (req, res, next) {
     const itemsPerPage = 10;
     const currentPage = parseInt(req.query.p) || 1;
 
-    api(req).get('/accounts/', {
+    api(req, { useCallback: false, json: true, version: 'v3' })
+        .get('/account', {
             qs: {
-                username: {
-                    $regex: _.escapeRegExp(req.query.q),
-                    $options: 'i'
-                },
-                $limit: itemsPerPage,
-                $skip: itemsPerPage * (currentPage - 1),
-                $sort: req.query.sort
+                type: 'username',
+                value: req.query.q,
+                limit: itemsPerPage,
+                skip: itemsPerPage * (currentPage - 1),
             }
-        }
-    ).then(data => {
-        const head = [
-            'ID',
-            'Username',
-            'Aktiviert',
-            'UserId',
-            ''
-        ];
-
-        const body = data.map(item => {
-            return [
-                item._id ||"",
-                item.username ||"",
-                item.activated ? '✔️' : '❌',
-                item.userId ||"",
-                getTableActions(item, '/accounts/')
+        })
+        .then(data => {
```

Comment:
Another place we could use await instead.

Reply:
I only changed the spacing and without automated tests I'm not gonna change to much code.